### ISSUE TITLE
Unable to install packages

### ISSUE THREAD

**Issue by envy2000** [clusterio/clusterio#615]
Platform: Windows 10

I am able to run npm, and can get through the first few prompts, but after the plugins selection step, I get an error as such
error Failed to install: spawn EINVAL

After some quick research, I found out about [this security patch](https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2), which seems to be the culprit.

With a bit more digging, I'm sure I could mess with my node version and get it to work, but this seems like a major enough issue that I thought I'd mention it here.

Specific details:
node version: 20.12.2.0
windows build: 19045.4291

**Comment by Danielv123** [comment]
This was resolved in #614 so will be fixed in the next release. There is a cli flag that should bypass the cve check (we are not affected) or you can use an older patch version of node.

**Comment by Danielv123** [comment]
See also #612

**Comment by envy2000** [comment]
yep, just downgraded node and it worked fine. I was originally using some much older version, but I was unable to setup the standalone with my version, and was prompted to update to 20.12.2. So I did. I kind of already suspected I just needed to downgrade when i opened the issue, but thought that I should make sure that attention had been brought to it because, again, it seemed like a pretty major issue.

I'm glad to know its already fixed, though.
